example/src/ModalStackInverted.js

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
```

Comment:
idk if we need a separate example for this. maybe add a toggle in an existing screen to flip this?

Reply:
It's great idea! We will test switching in the same point!

Reply:
Changed - now example instead of separate page have switch on modal page so you can dynamically interchange between vertical and vertical-inverted